news/2024/2024-06-27-osu-catch-world-cup-2024-finals-recap.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: "osu!catch World Cup 2024: Finals Recap"
+date: 2024-06-27 20:00:00 +0000
+---
+
+We have our podium finisher of this years world cup! Let's recap the Finals weekend.
```

Comment:
```suggestion
We have our podium finisher of this year's World Cup! Let's recap the Finals weekend.
```

Reply:
again "world cup" doesn't need to be capitalised on its own

Reply:
fixed